### ISSUE TITLE
PHP Fatal error: Uncaught Error: Call to a member function api_request()  ---- edd-updater.php:258

### ISSUE THREAD

**Issue by Curb72** [soflyy/oxygen-bugs-and-features#3299]
Oxygen wont save. Spinning wheel spinning and spinning and spinning... 

PHP Fatal error: Uncaught Error: Call to a member function api_request() on null in /home/837963.cloudwaysapps.com/fvmeztwtfd/public_html/wp-content/plugins/oxygen/component-framework/admin/updater/edd-updater.php:258

**Comment by Spellhammer** [comment]
@Curb72 Which version of Oxygen are you using?

**Comment by Spellhammer** [comment]
@Pino4 WordPress support for PHP 8.2+ is questionable at best. Try PHP 8.1.x and let me know if that resolves the issue.

**Comment by Kpudlo** [comment]
Reproduced and reported internally via https://github.com/soflyy/oxygen/issues/5059

**Comment by mickaelperrin** [comment]
I can confirm the issue. We trigger it by using WP CLI and the subcommand plugin.

For example:
```
wp plugin list
wp plugin install wordpress-importer --activate
```
triggers it

It looks like the `$oxygen_updater->edd_updater` is only registered through the `admin_init` hook.

However, `wp plugin` calls the hook `update_plugins_oxygenbuilder.com` which calls the `$oxygen_updater->edd_updater->api_request` function.

**Comment by Spellhammer** [comment]
This is fixed in Oxygen 4.6.1 RC 2, available at https://breakdance.com/portal. The final release will be available as an automatic update very soon.
